src/kernel-install/90-loaderentry.install

```diff
@@ -121,10 +122,11 @@ for initrd in "$@" "${KERNEL_INSTALL_STAGING_AREA}"/initrd*; do
 
     initrd_basename="${initrd##*/}"
     [ "$KERNEL_INSTALL_VERBOSE" -gt 0 ] && echo "Installing $ENTRY_DIR_ABS/$initrd_basename"
-    install -g root -o root -m 0644 "$initrd" "$ENTRY_DIR_ABS/$initrd_basename" || {
+    install -m 0644 "$initrd" "$ENTRY_DIR_ABS/$initrd_basename" || {
         echo "Error: could not copy '$initrd' to '$ENTRY_DIR_ABS/$initrd_basename'." >&2
         exit 1
     }
+    chown root.root "$ENTRY_DIR_ABS/$initrd_basename" || :
```

Comment:
the C programmer in mind kinda hates the fact that this means the file cannot be created atomically with the right ownership anymore